src/crawler/google_drive/week_review_count_com.google.android.apps.docs_in_en_.csv

```diff
@@ -0,0 +1,141 @@
+time,Frequency
+2014-03-16,2
+2014-09-14,1
+2015-03-29,1
+2017-03-19,1
+2018-09-16,56
+2018-09-23,101
+2018-09-30,233
+2018-10-07,283
```

Comment:
Would your code calculate the metrics for separate ratings?

Reply:
The current code doesnt calculate count for seperate ratings